src/containers/ProfilePage.jsx

```diff
@@ -78,6 +79,11 @@ class ProfilePage extends React.Component {
         })
         .then(res => res.json())
         .then((res) => {
+          if (res.links.length === 0 && !res._defaultprofile) {
```

Comment:
The check `res.links.length === 0` is not triggered, so the message never gets displayed